packages/dataviews/src/view-table.js

```diff
@@ -344,8 +344,7 @@ function ViewTable( {
 	const columns = useMemo( () => {
 		const _columns = fields.map( ( field ) => {
 			const { render, getValue, ...column } = field;
-			column.cell = ( props ) =>
```

Comment:
We can access the view in the fields API directly, no need to pass it to the field through render. Same for `view-grid.js`. This change is not necessary for this PR to work.